website/next.config.js

```diff
@@ -1,8 +1,20 @@
 const withHashicorp = require('@hashicorp/platform-nextjs-plugin')
 const redirects = require('./redirects.next')
 
+// log out our primary environment variables for clarity in build logs
```

Comment:
This is in stable website now but not the original PR https://github.com/hashicorp/vault/pull/13159 so not sure whether we want it or not, for example.